components/learn/StartLearningSection.vue

```diff
@@ -342,3 +352,24 @@ export default class StartLearningSection extends Vue {
   }
 }
 </style>
+
+<style lang="scss">
+  .start-learning-section {
+      // override tab position
+    .cv-tab.bx--tabs {
+      display: flex;
+      justify-content: flex-end;
+      padding-bottom: 3rem;
```

Comment:
since `padding` is a spacing property, we can update this to use [Carbon's spacing token ](https://carbondesignsystem.com/guidelines/spacing/overview/#spacing-scale) of `$spacing-09`